docs/2/technical/data-format/overview.md

```diff
@@ -114,55 +107,49 @@ Here we will see one by one the different components, starting with the unique v
 "updaterId": "yourname@microshare.io"
 ```
 
-* `"_id"`
+* `"_id"` and `"id"`
 
 The Id of the data is its unique identifier. It is used by a Microshare® Technician to find a particular piece of data in the datalake.
 
 * `"checksum"`
 
-The checksum is another identifier also for other needs for Microshare®.
+The checksum is another identifier that Microshare® technicians use to find that peice of data in the datalake.
 
 * `"createDate"`
 
-CreateDate as its name indicates quite well when this piece of data was created.
+CreateDate details the time the peice of data was created.
```

Comment:
CreateDate is an ISO 8601 date/time stamp in the YYYY-MM-DDThh:mm:ss.sTZD pattern that represents the date/time of the data's introduction to the Microshare system.